### ISSUE TITLE
[Bug] Large Font Accessibility issue

### ISSUE THREAD

**Issue by masukomi** [darkreader/darkreader#12645]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this bug to avoid opening a duplicate.
- [X] I can reproduce this bug in a new, unmodified web browser profile with Dark Reader installed as the only extension.
- [X] I understand I need to use the [Broken Website Report template](https://github.com/darkreader/darkreader/issues/new?assignees=&labels=Broken+Website&template=broken-website-report.yml&title=%5BBroken+Website%5D+Replace+with+title) if this bug I am reporting occurs on a single website.

### Bug Description

When your browser is set to use large fonts (and a minimum font size) things start degrading quickly. 

The first screenshot shows what happens with the fonts at 20pt (my default). The second is at 30.
Note that the "only for this website" button is barely clickable in the first and non-existent in the second. 

I think the solution I'd /like/ to see is to have the window width expand horizontally and vertically to accommodate the contents instead of scroll bars. I'm betting that's easier said than done, but it's also what i'd prefer as a large font user. Constantly seeing and dealing with scroll bars that no-one else has to is really annoying unless there's an absolute need. 

### Website Address

https://darkreader.org/

### Steps To Reproduce

Increase your font size. 
Increase your MINIMUM font size. 
Click the Dark Reader icon. 

### Expected Behavior

as a low vision user I expect to be able to be shown, and be able to click on everything that a perfect vision user can see. I also expect text that is within the borders of a box to remain within the borders of a box when displayed with larger fonts.

### Actual Behavior

Depending on how large my font is i may not see an item, or it may be obscured by other items or the window edge such that it has a very small click target.  Additionally many buttons (ex. "Dev Tools" in the second screenshot) have their text overflowing the bounds of the boxes they reside within.

Note: as fine motor control & eyesight both have a tendency to degrade with age small click targets are relevant here. 

### Screenshots

![CleanShot 2024-04-27 at 10 07 33@2x](https://github.com/darkreader/darkreader/assets/2829/5f38b23d-a61c-43bc-a076-a7a2a9d5302a)
![CleanShot 2024-04-27 at 10 09 47@2x](https://github.com/darkreader/darkreader/assets/2829/8d5e30f5-5927-415e-bc0f-40179fb14e24)

### Operating System

macOS

### Web Browser name and version

Firefox 126.0b4

### Dark Reader version

 Version 4.9.83

### Additional Context

the web site doesn't matter. this is an accessibility problem with the extension itself.

**Comment by wndxlori** [comment]
As a person with degraded vision, I too experience this issue. Please fix.

**Comment by charmarkk** [comment]
Good catch @masukomi! Darkreader is an indispensable accessibility tool for many (I recommend it all the time in my line of work, with much praise for the extension and team), would love to see this fixed.
